### DISCUSSION TITLE
PR triggers two checks, one passes, the other stuck in `Expected`

### DISCUSSION THREAD

**Discussion by cesarcoatl** [Homebrew/discussions#3040] [Tap maintenance and Homebrew development] [answered]
I maintain a [tap](https://github.com/coatl-dev/homebrew-coatl-dev), and since November 2021 every time I create a PR two checks are triggered for `brew test-bot`.

The only changes that I've made are removing the bottles step, and removing ubuntu from the `os` matrix.

At some point I had the following for `os`:

```yml
        os: [macOS-latest]
```

And for a test I did, I switched to:

```yml
        os: [macos-latest]
```

Since then I keep seeing two checks:

* **brew test-bot / test-bot (macos-latest) (pull_request)** Successful in 12m 
* **test-bot (macOS-latest)** _Expected — Waiting for status to be reported_

This prevents me from enabling auto-merge because not all checks pass, and there is no way I can get rid of the second check.

If anyone cares to check, this is my [`tests.yml`](https://github.com/coatl-dev/homebrew-coatl-dev/blob/coatl/.github/workflows/tests.yml).

Thanks

**Comment by Bo98** [comment, marked as answer]
You probably have the other one set in:

Settings -> Branches -> Branch protection rules (edit) -> Require status checks to pass before merging

**Reply by cesarcoatl** [reply]
That was it! Thanks, @Bo98!
